lib/experimental/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -36,15 +36,23 @@ public static function get_theme_data( $deprecated = array(), $settings = array(
 		if ( null === static::$theme ) {
 			$theme_json_data = static::read_json_file( static::get_file_path_from_theme( 'theme.json' ) );
 			$theme_json_data = static::translate( $theme_json_data, wp_get_theme()->get( 'TextDomain' ) );
-			$theme_json_data = gutenberg_add_registered_webfonts_to_theme_json( $theme_json_data );
-			static::$theme   = new WP_Theme_JSON_Gutenberg( $theme_json_data );
+
+			$original_theme_json = new WP_Theme_JSON_Gutenberg( $theme_json_data );
+			gutenberg_register_webfonts_from_theme_json( $original_theme_json->get_settings() );
+
+			if ( is_admin() ) {
```

Comment:
Is the `is_admin()` check here enough? 🤔 
Should we also add a check for REST requests?
```suggestion
			if ( is_admin() || ( defined( 'REST_REQUEST' ) && REST_REQUEST ) ) {
```